# A simple vector of double data
float64[] data